### DISCUSSION TITLE
/posts/2023/08/18%E7%BB%99vitepress%E5%A2%9E%E5%8A%A0%E7%9F%AD%E9%93%BE%E6%8E%A5%E7%94%9F%E6%88%90%E5%8A%9F%E8%83%BD

### DISCUSSION THREAD

**Discussion by giscus** [Justin3go/justin3go.com#41] [Announcements]
# /posts/2023/08/18%E7%BB%99vitepress%E5%A2%9E%E5%8A%A0%E7%9F%AD%E9%93%BE%E6%8E%A5%E7%94%9F%E6%88%90%E5%8A%9F%E8%83%BD

坚持深耕技术领域的T型前端程序员, 关注独立开发与AI，喜欢Vuejs、Nestjs, 还会点Python、搜索引擎、NLP、Web3、后端

https://justin3go.com/posts/2023/08/18%E7%BB%99vitepress%E5%A2%9E%E5%8A%A0%E7%9F%AD%E9%93%BE%E6%8E%A5%E7%94%9F%E6%88%90%E5%8A%9F%E8%83%BD

**Comment by ATQQ** [comment]
这个功能不错，可以抄一个 分离成插件（(#^.^#)）

**Reply by Justin3go** [reply]
才注意到这个评论，快抄快抄，不过还有一个问题没有解决，就是短链接的SEO优化没做，比如一些meta信息

**Comment by MWTJC** [comment]
极好的，跪一个
我没怎么接触过node，我试图在使用[autosidebar](https://vitepress-sidebar.cdget.com/zhHans/)的情况下（懒），在自己的vitepress上也照葫芦画个瓢，发现有地方跑不通，但发现我似乎没能力对theme里的index.js及其加载vue组件的过程进行调试：idea打断点没反应，vscode直接就是“断点not bind”，浏览器控制台也只有句`Uncaught TypeError: Class extends value undefined is not a constructor or null`，
佬这边有没有啥资料或存货能讲明这vitepress开发调试环境怎么搭建，俺想拿去看看

**Reply by Justin3go** [reply]
没研究过，目前还是古法-日志大法调试的，忏愧
